.goreleaser.yml

```diff
@@ -0,0 +1,25 @@
+builds:
+  - id: "carbon-relay-ng"
+    main: ./cmd/carbon-relay-ng
+    binary: carbon-relay-ng
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
```

Comment:
amd64 only arch supported today because we ran into issues when compiling to arm due to our metrictank dependency